products/bash/setup-demos.sh

```diff
@@ -155,16 +162,58 @@ function merge_addon {
   echo ${ADDON_ARRAY_JSON}
 }
 
+function update_status() {
+  MESSAGE=${1}
+  TYPE=${2}        # resource type
+  RESULT_CODE=${3} # success(0), error(1)
+  TIMESTAMP=${4}
```

Comment:
Could we avoid passing this in? Just make this function calculate the timestamp?